bin/cpan-install-dist-deps

```diff
@@ -3,11 +3,11 @@
 set -eux
 
 if [[ -e cpanfile ]];then
-    cpm install -g --cpanfile cpanfile --with-recommends --with-suggests --show-build-log-on-failure "$@"
+    cpm install -g --cpanfile cpanfile --reinstall --with-recommends --with-suggests --show-build-log-on-failure "$@"
```

Comment:
I don't think I want `--reinstall` by default as the intent of this library was to use it in tandem with Docker images on a nightly build that should have everything pre-installed. I think if people want to `--reinstall`, they should probably call `bin/cpan-install-dist-deps` directly and pass that in explicitly.

Reply:
see for example job https://github.com/happy-barney/perl-Test-YAFT/actions/runs/5672790143

what about introducing new ENV variable, eg: `PERL_CPM_FORCE_REINSTALL`  ?

Reply:
Maybe we could even do something like:

`PERL_CPM_OPTS="--foo bar --baz qux` and then split the var on spaces in order to pass arbitrary args to `cpm`. Or use some other delimiter to split on.